src/overlays/actors/ovl_En_Cow/z_en_cow.c

```diff
@@ -5,9 +5,8 @@
  */
 
 #include "z_en_cow.h"
-#include "objects/object_cow/object_cow.h"
 
-#define FLAGS 0x00000009
+#define FLAGS (ACTOR_FLAG_8 | ACTOR_FLAG_1)
```

Comment:
```suggestion
#define FLAGS (ACTOR_FLAG_1 | ACTOR_FLAG_8)
```
Could you swap the order for these? OoT has these defines in ascending order and since some (not all but some) flags match up with OoT, it'd be good to have things line up when these eventually get named